docker/r10k/Dockerfile

```diff
@@ -62,7 +67,6 @@ RUN chmod a+x /adduser.sh /docker-entrypoint.sh && \
     mv "$supercronic" "/usr/local/bin/${supercronic}" && \
     ln -s "/usr/local/bin/${supercronic}" /usr/local/bin/supercronic
 
-USER puppet
-WORKDIR /home/puppet
```

Comment:
Add comment above this with content `# hadolint ignore=DL3000` to prevent a travis failure on Dockerfile linting.

Reply:
Done